etc/profile-m-z/youtube.profile

```diff
@@ -0,0 +1,38 @@
+# Firejail profile for youtube
+# Description: Unofficial electron based desktop warpper for YouTube
+# This file is overwritten after every install/update
+# Persistent local customizations
+include youtube.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/Youtube
+
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-shell.inc 
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/Youtube
+whitelist ${HOME}/.config/Youtube
+include whitelist-common.inc 
+include whitelist-runuser-common.inc 
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+nou2f
```

Comment:
YT has 2FA AFAIK, this wrapper too?

Reply:
It's a wrapper, but it should work? I forgot...I don't use it.